pkg/crc/machine/virtualmachine.go

```diff
@@ -32,6 +32,14 @@ func (err *MissingHostError) Error() string {
 	return fmt.Sprintf("no such libmachine vm: %s", err.name)
 }
 
+func (err *MissingHostError) Is(target error) bool {
+	var x *MissingHostError
+	if errors.As(target, &x) && x.name == err.name {
+		return true
+	}
+	return false
+}
```

Comment:
This probably addresses some of the remarks I made before, but some explanations are always nice to have :)